File: java/potion.java
public class potion {
    private int lvlreq;
    private String type;
    private int value;
    private int turns;
    private int cost;
    public potion(int lr, String t, int i, int duration, int c) {
        lvlreq = lr;
        type = t;
        value = i;
        turns = duration;
        cost = c;
    }
    //GETTERS
    public int getLvlReq() {return lvlreq;}
    public String getType() {return type;}
    public int getValue() {return value;}
    public int getTurns() {return turns;}
    public int getCost() {return cost;}
}

File: game/potion.js
export class potion {
    lvlreq;
    type;
    value;
    turns;
    cost;
    potion(lvlreq, type, value, turns, cost) {
        this.lvlreq = lvlreq;
        this.type = type;
        this.value = value;
        this.turns = turns;
        this.cost = cost;
    }
    //GETTERS
    getLvlReq() {return lvlreq;}
    getType() {return type;}
    getValue() {return value;}
    getTurns() {return turns;}
    getCost() {return cost;}
}
